module-1/pandas-project/your-code/Jupyter_pandas_project.ipynb

```diff
@@ -0,0 +1,6276 @@
+{
```

Comment:
1. It is important to include a brief description of the dataset you are going to use and an overview of the main steps you have followed in your analysis. 
2. Remember to print/display only the first rows of the dataframes. It's not necessary to print the complete dataframe. If you want to check some stuff in your dataframe you will need to check through coding. 
3. In [108]: You say 
#### He cambiado los nombres de la table de 2017 para que sean exactamente iguales que los de la data de 2016 y coincidan###, I guess that all the fields contains the same type of data and it's only a matter of column labels. Is this the case? This kind of things must be explained for better comprehension. 
4. You say, #### Realizo unas datas nuevas con las columnas que me van a ser utiles par realizar la conclsion#### . Actually it is the other way around, you slice the dataframe by the columns you need to run the analysis you have planned in advance. 
5. Why have you used an outer as the way of doing the merge 
6. You say, ##### Conclusion: Con este analisis podemos concluir que el nivel de felicidad para algunos paises depende de algun modo al nivel de felicidad del pais. Es decir, el ranking esta liderado por paises que se situan en una posicion high en funcion a la aportacion de GDP a la felicidad de las personas. pero por otro lado tambien podemos encontrar casos exclusivos como Costa Rica y Brazil, los cuales a pesar de que su aportacion de gdp al nivel de Happines es menos que muchos paises se encuentran en un posicion bastante alta lo que indica que para ellos el GDP no es un factor determinante para su felicidad
##### Tambin podemos obsevar como los paises menos felices (los ultimos del ranking) la grana mayoria cuentan con una aportacion low por parte del GDP por lo que de algun modo podemos definirlo como determinante para ello. In the real world the conclusions of your analysis must be fully supported by your data, your analysis and your code. 
7. Really good try your last step in the file!!, I guess this is not the best way of doing what I guess you pretend to try.  Get back to me if you are interested on how you can approach.